articles/active-directory/devices/troubleshoot-hybrid-join-windows-legacy.md

```diff
@@ -38,13 +38,15 @@ This article provides you with troubleshooting guidance on how to resolve potent
 **What you should know:** 
 
 - Hybrid Azure AD join for downlevel Windows devices works slightly differently than it does in Windows 10. Many customers do not realize that they need AD FS (for federated domains) or Seamless SSO configured (for managed domains).
+- Seamless SSO doesn't work in private browsing mode on Firefox and Microsoft Edge browsers. It also doesn't work on Internet Explorer if the browser is running in Enhanced Protected mode. For the next version of Microsoft Edge based on Chromium, it will not work in InPrivate and Guest mode by design.
```

Comment:
We dont talk about future products or features in docs. Remove "For the next version of Microsoft Edge based on Chromium, it will not work in InPrivate and Guest mode by design." You are already saying it doesnt work in Edge in the first sentance.